install-php-extensions

```diff
@@ -1192,6 +1192,7 @@ installRequiredPackages() {
 	case "$DISTRO" in
 		alpine)
 			apk add $IPE_APK_FLAGS $PACKAGES_PERSISTENT_NEW $PACKAGES_VOLATILE
+			apk upgrade
```

Comment:
Oh wow, who would have though about that! (The reason is linked via the references issue, but for easier access -> https://gitlab.alpinelinux.org/alpine/aports/-/issues/12833#note_167767 )